popup_offer/popup_offer.info.yml

```diff
@@ -1,10 +0,0 @@
-name: Popup Offer promotion functionality
-description: Provides a custom functionality for the promotion offers
-core: 8.x
-type: module
-dependencies:
-  - commerce
-  - commerce_cart
-  - commerce_checkout
-  - commerce_promotion
-  - modal_page
```

Comment:
why do we need this dependency? I'm not sure if it is using anywhere

Reply:
also there is no new line but it should be according to coding standards

Reply:
that module provided me a popup, just not to write own js and css I used that module theme function. New line agree.